libbzip2-rs-sys-cdylib/Cargo.toml

```
@@ -0,0 +1,36 @@
+[package]
+name = "libz-rs-sys-cdylib"
+version = "0.0.0"
+edition = "2021"
+readme = "README.md"
+license = "Zlib"
```

BZip2 is not under the Zlib license. The official SPDX license identifier seems to be:

```suggestion
license = "bzip2-1.0.6"
```